code/modules/martial_arts/martial.dm

```diff
@@ -6,6 +6,7 @@
 	var/temporary = 0
 	var/datum/martial_art/base = null // The permanent style
 	var/deflection_chance = 0 //Chance to deflect projectiles
+	var/reroute_deflection = FALSE // Can it reflect projectiles in a random direction?
```

Comment:
Sound effect is good. Reflection is OP and I will object to this PR until reflection is removed.

Reply:
I'd rather this be closed than the reflect removed, I understand this may be me being stubborn but I don't think i'll change my mind on this. I'm fully willing to accept this being closed if the reflect isn't wanted